<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettingsDialog</class>
 <widget class="QDialog" name="CameraSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TaurusWidget" name="taurusWidget">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Image Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TaurusValueComboBox" name="imageTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Trigger Mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="TaurusValueComboBox" name="triggerModeComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Exposure (ms)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="MAXLineEdit" name="expTimeLineEdit">
        <property name="model">
         <string/>
        </property>
        <property name="useParentModel">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Gain (dB)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="MAXLineEdit" name="gainLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaurusValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueComboBox</class>
   <extends>QComboBox</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>MAXLineEdit</class>
   <extends>TaurusValueLineEdit</extends>
   <header>maxwidgets.input</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
